Resources/Locale/en-US/nuke/nuke-component.ftl

```diff
@@ -32,9 +32,6 @@ nuke-user-interface-second-status-cooldown-time = WAIT: {$time}
 
 ## Nuke labels
 nuke-label-nanotrasen = NT-{$serial}
-
-# do you even need this one? It's more funnier to say that
```

Comment:
keep the comments. even though we use the string  the comment should probally stay.